Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
Ok for `skalibs`, but the thing is that `execline` is useful on its own. What would this mean: i remove the `skalibs` formula, but keep the static linking of s6 against `execline`?